Update links for support (#3681)
SEED-platform/seed: docs/source/help.rst

Help
====

For SEED Platform Users
^^^^^^^^^^^^^^^^^^^^^^^

Please visit our website for information, tutorials, and documentation to help you learn how to use SEED.

https://seed-platform.org

> The SEED User's forum is where you can review user announcements, workflow questions, and join to connect with other users.

Author: ```suggestion
The SEED Users forum is where you can review user announcements, workflow questions, and join to connect with other users.
```

https://lists.buildingenergytools.org/g/SEEDusers/topics

For direct help on a specific problem, please fill out a request on the building data tools website help desk:

https://buildingdata.energy.gov/#/help-desk

For SEED Platform Developers
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The open-source code is available on the Github organization SEED-Platform and contains various repositiories for the different components of the platform such as the main SEED application, a Python SEED client to communicate to SEED's API, and various example datasets.

Author: ```suggestion
The open-source code is available on the GitHub organization SEED-Platform and contains various repositories for the different components of the platform such as the main SEED application, a Python SEED client to communicate to SEED's API and various example datasets.
```

https://github.com/SEED-platform

> The SEED developer's forum contains various topics and joining enables you to connect with other developers. It is recommended to join this forum to sumbit developer questions, featuers requests, and report issues as needed. Also, submitting issues on Github is encouraged.

Author: ```suggestion
The SEED developers forum contains various topics and joining enables you to connect with other developers. It is recommended to join this forum to submit developer questions, features requests, and report issues as needed. Also, submitting issues on GitHub is encouraged.
```
